### PR TITLE
Feature/sc 30332/prepare suggested api contract changes for airport transfer

### PR DESCRIPTION
**Pull request by kalaondr**
https://app.shortcut.com/daytrip/story/30332/prepare-suggested-api-contract-changes-for-airport-transfers

### CODE REVIEW

File: source/includes/trip-api.md [hunk 1]
```diff
@@ -895,6 +927,81 @@ curl https://api.staging.mydaytrip.net/partners/v3/trip/details/bookingId
...
+{
+   "bookingReference": "72R537",
+   "status": "Confirmed",
+   "createdAt": "2022-12-05T18:00:00Z",
+   "passengersCount": 3,
+   "currency": "EUR",
+   "meetingPosition": {
+      "lat": 46.067648,
+      "lon": 7.775185,
+      "description": "the train station in Tasch",
```
Comment (woothu) [inline source/includes/trip-api.md:955]: We don't return googleAdress as well?

Comment (kalaondr) [reply source/includes/trip-api.md:955]: I will ask, for the meeting positions I checked it was not set so it would have to be optional.

Comment (woothu) [reply source/includes/trip-api.md:955]: While setting restricted areas on FE you need to add googleAdress, otherwise it cannot be saved

Comment (kalaondr) [reply source/includes/trip-api.md:955]: For some positions inside the airports for example I don't think it should be mandatory.

I think if partner wants to show the map it's better to use the coordinates. The optional Google address might add confusion.

Comment (woothu) [reply source/includes/trip-api.md:955]: You can still adjust coordinates after filling in google address for precise pin

Comment (woothu) [reply source/includes/trip-api.md:955]: Oh you mean about the partner, this is tricky thing.

Comment (kalaondr) [reply source/includes/trip-api.md:955]: Yeah, if coordinates can be adjusted without changing the Google address then we should definitely return just the coordinates.

**Review by jowczar — APPROVED** [review]

**Pull request merged**